src/ci/docker/host-x86_64/dist-x86_64-linux/build-gccjit.sh

```diff
@@ -1,7 +1,9 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
 set -ex
 
+source /tmp/shared.sh
```

Comment:
Why didn't you put this in the same folder as the build-gccjit script?

Reply:
Mostly because I was first trying to make it work. Cleanup time!